### ISSUE TITLE
Drag problem in sub-component

### ISSUE THREAD

**Issue by pkuwwt** [michitaro/vue-window#4]
I tried vue-window in [iview-admin](https://github.com/iview/iview-admin) as a sub-component, i.e. associate it to a vue-router path. However, there is a problem.

When I drag the window, the start-drag-position is not the position I start clicking. In specific, offsetx = width(left-panel), offsety = height(top-menu-bar). So that the window always jump toward bottom-right when I start dragging.

I am sorry I cannt reproduce a minimal example for now. Do any one know where is the problem?

**Comment by michitaro** [comment]
Sorry for the late reply. 

I think you use vue-window inside a non-static positioned element, right?
The problem is caused by something around that.

Anyway, I cannot inspect without any reproduction code.

**Comment by pkuwwt** [comment]
yes, the component is dynamically created only after the route path is accessed.

**Comment by michitaro** [comment]
Thank you for the jsfiddle.
I'll inspect it.

**Comment by egorovd** [comment]
Thank you.
Is it possible to add a theme management for 'title' ?

**Comment by egorovd** [comment]
http://jsfiddle.net/xqth7y6s/3/
Can be otherwise.
But then the top toolbar can cover the window title.
